### DISCUSSION TITLE
Quantity based discounts

### DISCUSSION THREAD

**Discussion by krzysztofwolski** [saleor/saleor#14432] [Ideas]
## Problem

Would like to give customers a better price based on ordered quantity:
- When buying 1 piece, piece price would be $10
- When buying 10 pieces, piece price would be $8 (20% discount)
- When buying 50 pieces, piece price would be $7 (30% discount)

## 1. General Assumptions

- Multiple discount steps can be defined
- Discounts can be combined with others

**Comment by DustinBracy** [comment]
Definitely could use this!  I have a production use case for this right now: we're stuck either trying to sync inventory between variants (i.e. singles/packs), or manually adding discounts on bulk singles.

**Comment by timuric** [comment]
We are in process of building new promotion engine which would cover this use case, you can track promotion related work using this label
https://github.com/saleor/saleor/issues?q=is%3Aissue+is%3Aopen+label%3Apromotions
